tests/storages_tests/rdb_tests/test_storage.py

```diff
@@ -89,6 +89,38 @@ def test_engine_kwargs():
         create_test_storage(engine_kwargs={'wrong_key': 'wrong_value'})
 
 
+def test_set_default_engine_kwargs_for_mysql():
```

Comment:
A small nit but it looks like you could parameterize it with `(<url>, <initial engine_kwargs>, <expected boolean>)`

Reply:
You're right. I'll parameterize it. Thanks!

Reply:
I fixed it in https://github.com/optuna/optuna/pull/806/commits/453711e333a1d9910417d6cc12e6bfb82de19037. I extracted the test cases for other RDB into `test_set_default_engine_kwargs_for_mysql_with_other_rdb` because such cases cannot handled with `(<url>, <initial engine_kwargs>, <expected boolean>)`.

Reply:
Ah, I though that for instance a `None` in `<expected boolean>` could have a special meaning but it's really a small detail. LGTM!